docs/HOW_TO_UPGRADE.md

```diff
@@ -0,0 +1,148 @@
+# HOW TO UPGRADE PARTICIPA.GENCAT.CAT
+
+## Steps to follow:
+In order to update the application correctly, the points commented in this file must be followed:
+
+Decidim are releasing new versions continuously. In order to get the latest one, update the dependencies:
+
+We recommend upgrading version to version to prevent errors.
+
+### Steps
+
+1. Go to the Gemfile and upgrade the decidim version to the next one stable.
+
+2. Go to the console and run
+```console
+bundle update decidim
+```
+
+3. Make sure you get all the latest migrations:
+```console
+bin/rails decidim:upgrade
+bin/rails db:migrate
+```
+
+4. Go to the changelog of the specific branch you are updating. For example, https://github.com/decidim/decidim/blob/master/CHANGELOG.md. And review each of the new developments introduced, and apply the new changes. If there are some new locales added or changed, take in mind, that until the locale :oc is added to the Decidim, you must add the new Occitan translations to the application.
+
+5. also, you can make sure new translations are complete for all languages in your application doing this:
+First, you must define which version of Decidim you are updating by saying the branch in the application.yml, as: `TARGET_BRANCH: "master"`<br/>
+Then execute:
+```console
+bin/rails decidim:check_locales
```

Comment:
maybe we should use `TARGET_BRANCH=master bin/rails decidim:check_locales` instead of TARGET_BRANCH in application.yml?